security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -668,6 +669,16 @@
                 <dns_gcloud_key type="TextField">
                     <Required>N</Required>
                 </dns_gcloud_key>
+                <dns_googledomains_access_token type="TextField">
+                    <Required>N</Required>
+                </dns_googledomains_access_token>
+                <dns_googledomains_api_endpoint type="TextField">
+                    <Required>N</Required>
+                    <default>https://acmedns.googleapis.com/v1/acmeChallengeSets</default>
```

Comment:
I don't think we should make the API endpoint configurable. It adds a config option that all users have to change, if Google decides to change the API endpoint URL. We should [rely on acme.sh to provide the correct URL](https://github.com/acmesh-official/acme.sh/blob/0d25f7612bf37a42d9c4fcb1abc493b2a5e495c3/dnsapi/dns_googledomains.sh#L8) instead.

So I suggest to remove the parameter `dns_googledomains_api_endpoint`.

Reply:
ACK, that makes sense.

I was adding it for completeness, but ideally it would be a hidden or advanced option but I don't know what's possible here. 

Made a note in a comment in the php file that it is technically configurable should it ever turn out to be useful to add it.